ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -235,6 +245,7 @@ private static AccountService getHelixAccountService(String zkServer, String sto
         String.valueOf(zkSessionTimeoutMs));
     helixConfigProps.setProperty(HelixAccountServiceConfig.ZK_CLIENT_CONNECT_STRING_KEY, zkServer);
     helixConfigProps.setProperty(HelixPropertyStoreConfig.HELIX_PROPERTY_STORE_PREFIX + "root.path", storePath);
+    helixConfigProps.setProperty(HelixAccountServiceConfig.BACKUP_DIRECTORY_KEY, backupDir);
```

Comment:
Do we need to specify default `backDir` in `HelixAccountServiceConfig`? Currently, the default value is empty string.

Reply:
This feature is disabled by default. That is why the default is empty string